### DISCUSSION TITLE
عدم امکان درج نام فایل به فارسی

### DISCUSSION THREAD

**Discussion by imanjahanyan** [kianahmadian/python-bale-bot#88] [Q&A] [answered]
با سلام
وقتی فایل رو برای ارسال انتخاب می کنیم و نامش رو به صورت فارسی قرار میدیم،
نام فایل در پیام ارسالی به صورت unicode درج میشه

            with open(pdf_path, 'rb') as f:
                file = attachments.InputFile(f.read())
                name = basename(pdf_path)
                file.file_name = name
                await client.send_document(chat_id=answer_obj.chat_id, document=file, reply_to_message_id=answer_obj.message_id)
            
           ![Screenshot 2024-03-09 144757](https://github.com/python-bale-bot/python-bale-bot/assets/64071186/7da4b7c0-b822-47cf-b265-9e0dbf49ecbe)

**Comment by imanjahanyan** [comment]
with open(pdf_path, 'rb') as f:
            file = attachments.InputFile(f.read())
            name = basename(pdf_path)
            file.file_name = name
            await client.send_document(chat_id=answer_obj.chat_id, document=file, reply_to_message_id=answer_obj.message_id)
        
![Screenshot 2024-03-09 144757](https://github.com/python-bale-bot/python-bale-bot/assets/64071186/92e28ad1-d52c-45f9-a929-daa99f8f8838)

**Comment by kian-ahmadian** [comment, marked as answer]
سلام عزیز وقت بخیر
**از گزارش تون سپاسگزارم؛ به زودی این مورد و بررسی و نتیجه رو خدمت شما اطلاع خواهیم داد**

همچنین به زودی و در نسخه جدید کتابخانه بخش `attachments` دچار تغییراتی خواهد شد؛ بنابراین پیشنهاد می‌کنیم برای دریافت اطلاعات بیشتر کتابخانه را دنبال کرده و یا از طریق بخش بروزرسانی ها در سایت اطلاعات بیشتر در این رابطه را همواره بررسی نمایید
